### PR TITLE
P0 - Alert and avatar components

### PR DESCRIPTION
**Pull request by bharati-21**
- Created and styled alert component. Added it to the docs page.
  - The alert component has 2 variations - simple alerts and simple alerts with a close icon.
  - Used JavaScript for close alert functionality. 
- Created and styled avatar component. Added it to the docs page.
  - The avatar component has 4 variations - circle image avatar, square image avatar, circle text avatar, and square text avatar

### CODE REVIEW

**Comment by pnchinmay** [comment]
Hey @bharati-21, the preview link isn't working bcoz `./Components/alerts/index.html` isn't linked in `./index.html` present in root folder.
Also, I would suggest to name it as `./Components/alerts/alert.html`

image
Margin isn't present on the right side. Also, keep the width of these alerts max upto half width of the page.
**Colors are awesome.** Those close buttons are working great

**Review by pnchinmay — APPROVED** [review]

**Comment by pnchinmay** [comment]
Avatars look great

**Review by pnchinmay — APPROVED** [review]

**Comment by bharati-21** [comment]
> Hey @bharati-21, the preview link isn't working bcoz `./Components/alerts/index.html` isn't linked in `./index.html` present in root folder. Also, I would suggest to name it as `./Components/alerts/alert.html`
> 
> image Margin isn't present on the right side. Also, keep the width of these alerts max upto half width of the page. **Colors are awesome.** Those close buttons are working great.


----

Hey, @pnchinmay  thank you for your review.

Actually, the docs page is not the one that's previewed by netlify here. It's the landing page (which I'm yet to add). 
You can look at the docs page here https://deploy-preview-1--elixir-ui.netlify.app/docs/ . I had mentioned it when I dropped the PR request.

As for the `alert/index.html`, it was just a page I added to visualize my alerts outside of docs. But I shall add margin and keep width up to half the width.

Thanks. Please do check the docs page once https://deploy-preview-1--elixir-ui.netlify.app/docs/ so that I can understand what to remove/ add to the docs.

**Comment by pnchinmay** [comment]
> Actually, the docs page is not the one that's previewed by netlify here. It's the landing page (which I'm yet to add). You can look at the docs page here https://deploy-preview-1--elixir-ui.netlify.app/docs/ . I had mentioned it when I dropped the PR request.
> 
> As for the `alert/index.html`, it was just a page I added to visualize my alerts outside of docs. But I shall add margin and keep width up to half the width.
> 
> Thanks. Please do check the docs page once https://deploy-preview-1--elixir-ui.netlify.app/docs/ so that I can understand what to remove/ add to the docs.

Sorry about that. I should've read the msg.
Docs site looks great. Good to go

**Comment by bharati-21** [comment]
> > Actually, the docs page is not the one that's previewed by netlify here. It's the landing page (which I'm yet to add). You can look at the docs page here https://deploy-preview-1--elixir-ui.netlify.app/docs/ . I had mentioned it when I dropped the PR request.
> > As for the `alert/index.html`, it was just a page I added to visualize my alerts outside of docs. But I shall add margin and keep width up to half the width.
> > Thanks. Please do check the docs page once https://deploy-preview-1--elixir-ui.netlify.app/docs/ so that I can understand what to remove/ add to the docs.
> 
> Sorry about that. I should've read the msg. Docs site looks great. Good to go

No problem!
Thank you for your time and review! :D

File: Components/alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+:root {
+    --secondary-bg-color: #c3dbff;
+    --warning-bg-color: #fce6cf;
+    --error-bg-color: #ffd0d0;
+    --success-bg-color: #a8fcd6;
+}
```
Comment (yashpatel1797) [inline Components/alerts/alert.css:6]: If this is common colour then you can include it in base CSS but if specific to only alert then fine.

File: Components/alerts/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline Components/alerts/index.html:6]: As per Smile Gupta PR review, you can add a name in the meta tag.

File: docs/index.html [hunk 3]
```diff
@@ -69,9 +69,256 @@ <h4 class="sidebar__head">
...
+                                <p class="alert-text">This is an error alert</p>
+                            </div>
+                        </div>
+                        <div class="code-snippet-container">
+                            <p>Paste  the code below to use the alerts with different variations</p>
+
+                            <div class="code-display">
+                                <iframe
+                                    src="https://carbon.now.sh/embed/uUNSWLE4IDf7K8vCVq6y"
+                                    style="width: 100%;"
```
Comment (yashpatel1797) [inline docs/index.html]: Avoid writing inline CSS.

**Review by yashpatel1797 — COMMENTED** [review]
1- Rename avatars/index.html to avatars/avatar.html. 
2- The naming convention is good.
3- Design is clear and colour choice is also good.

File: Components/alerts/alert.css (hunk 1 above)
Comment (bharati-21) [reply Components/alerts/alert.css:6]: This is specific to alert. In the future, if it becomes common I shall move it. 👍🏽

**Comment by bharati-21** [comment]
> 1- Rename avatars/index.html to avatars/avatar.html. 2- The naming convention is good. 3- Design is clear and colour choice is also good.

1. The index.html file is one I created for visualizing the components alone, outside of the component library. I'm not using it on the website. But yes, I shall change the name or remove it, since it's causing a lot of confusion.
2. Thank you!
3. Thank you!

File: Components/alerts/index.html (hunk 2 above)
Comment (bharati-21) [reply Components/alerts/index.html:6]: Yes, I shall add it 👍🏽

File: docs/index.html (hunk 3 above)
Comment (bharati-21) [reply docs/index.html]: Yes, will remove this.

**Comment by bharati-21** [comment]
> 1- Rename avatars/index.html to avatars/avatar.html. 2- The naming convention is good. 3- Design is clear and colour choice is also good.

@yashpatel1797, I made the changes you asked for. Can you please check them once and approve them?
Thanks.

**Comment by yashpatel1797** [comment]
> > 1- Rename avatars/index.html to avatars/avatar.html. 2- The naming convention is good. 3- Design is clear and colour choice is also good.
> 
> @yashpatel1797, I made the changes you asked for. Can you please check them once and approve them? Thanks.



> > 1- Rename avatars/index.html to avatars/avatar.html. 2- The naming convention is good. 3- Design is clear and colour choice is also good.
> 
> 1. The index.html file is one I created for visualizing the components alone, outside of the component library. I'm not using it on the website. But yes, I shall change the name or remove it, since it's causing a lot of confusion.
> 2. Thank you!
> 3. Thank you!

@bharati-21
1> Actually I see that you have mentioned that but I haven't read it, a mistake from my side.

**Review by yashpatel1797 — APPROVED** [review]

**Comment by bharati-21** [comment]
> > > 1- Rename avatars/index.html to avatars/avatar.html. 2- The naming convention is good. 3- Design is clear and colour choice is also good.
> > 
> > 
> > @yashpatel1797, I made the changes you asked for. Can you please check them once and approve them? Thanks.
> 
> > > 1- Rename avatars/index.html to avatars/avatar.html. 2- The naming convention is good. 3- Design is clear and colour choice is also good.
> > 
> > 
> > 
> > 1. The index.html file is one I created for visualizing the components alone, outside of the component library. I'm not using it on the website. But yes, I shall change the name or remove it, since it's causing a lot of confusion.
> > 2. Thank you!
> > 3. Thank you!
> 
> @bharati-21 1> Actually I see that you have mentioned that but I haven't read it, a mistake from my side.

No problem! 
Thank you for your time and review 😄

File: Components/alerts/index.html (hunk 2 above)
Comment (bharati-21) [reply Components/alerts/index.html:6]: Added desc and author meta tags.

**Pull request merged**